RecoHI/HiTracking/python/HIBestVertexSequences_cff.py

```diff
@@ -15,7 +15,8 @@
 )
 
 # best vertex sequence
-bestHiVertex = cms.Sequence( hiBestAdaptiveVertex * hiSelectedPixelVertex ) # vertexing run BEFORE tracking
+bestHiVertexTask = cms.Task( hiBestAdaptiveVertex , hiSelectedPixelVertex ) # vertexing run BEFORE tracking
+bestHiVertex = cms.Sequence( bestHiVertexTask ) # vertexing run BEFORE tracking
```

Comment:
This was only needed in RecoHI/HiTracking/python/HIPixelVertices_cff.py: sequence not needed any more now, please remove